addOns/ascanrulesBeta/CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [47] - 2023-07-20
 ### Added
-- The Source Code Disclosure - File Inclusion now includes example alert functionality for documentation generation purposes (Issue 6119).
+- The Source Code Disclosure - File Inclusion and HttpOnlySiteScanRule now includes example alert functionality for documentation generation purposes (Issue 6119).
```

Comment:
```suggestion
- The Source Code Disclosure - File Inclusion and HTTP Only Site rules now include example alert functionality for documentation generation purposes (Issue 6119).
```

Reply:
It was not the final push, this needs to be moved to Unreleased.